plio/settings.py

```diff
@@ -309,3 +309,9 @@
 EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD")
 
 SMS_DRIVER = os.environ.get("SMS_DRIVER")
+
+# file storage for uploaded images
+DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+AWS_STORAGE_BUCKET_NAME = os.environ.get("AWS_STORAGE_BUCKET_NAME")
+AWS_QUERYSTRING_AUTH = False
```

Comment:
<img width="682" alt="Screenshot 2021-06-13 at 9 38 54 PM" src="https://user-images.githubusercontent.com/13185699/121814641-c3111b80-cc8f-11eb-9071-d899b7ad42e9.png">
false is for only public s3 buckets right? We shouldn't have it public right?

Reply:
If we don't have a public access enabled for the images, the images will not load on the client, or any browser even. Earlier the bucket was not public, it had no policy for any public user to read the files. And testing during that, I was not able to load the files on the plio

Reply:
bucket level private, file level public